Source/ARTRealtimeChannel.m

```diff
@@ -1145,7 +1162,7 @@ - (BOOL)history:(ARTRealtimeHistoryQuery *)query callback:(ARTPaginatedMessagesC
     return [_restChannel history:query callback:callback error:errorPtr];
 }
 
-- (void)startDecodeFailureRecoveryWithChannelSerial:(NSString *)channelSerial error:(ARTErrorInfo *)error {
+- (void)startDecodeFailureRecoveryWithChannelSerial:(ARTErrorInfo *)error {
```

Comment:
Since `channelSerial` param was removed, name should reflect this:
`startDecodeFailureRecoveryWithErrorInfo:`

Reply:
573d5ebf5364bea59af5a1c31fbf6710b3c4e1ff